beets/dbcore/db.py

```diff
@@ -1039,12 +1039,11 @@ def _fetch(self, model_cls, query=None, sort=None):
         flex_sql = ("""
             SELECT * FROM {0} WHERE entity_id IN
                 (SELECT id FROM {1} WHERE {2});
-            """.format(
-                model_cls._flex_table,
-                model_cls._table,
-                where or '1',
-            )
-        )
+            """.format(model_cls._flex_table,
+                       model_cls._table,
+                       where or '1',
+                       )
+                    )
```

Comment:
Weird—I think this conformant indentation is now _less_ clear, somehow? Maybe it would look less strange if we just put the closing parentheses right after the `'1'` instead of on their own lines.